lib/bootloader_setup.pm

```diff
@@ -353,7 +353,7 @@ sub get_bootmenu_console_params {
 
     # See bsc#1011815, last console set as boot parameter is linked to /dev/console
     # and doesn't work if set to serial device. Don't want this on some backends.
-    push @params, "console=tty" unless (get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/);
+    push @params, "console=tty" unless (get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/ || check_var('PUBLIC_CLOUD', 1));
```

Comment:
from what I understand just this change without changing FS suppose to fix the problem . or am I missing something ?

Reply:
It should, but the main underlying issue remains. I don't see any downsides of switching to the default filesystem for SLES, which should solve the issue where it actually occurs.

Reply:
```suggestion
    push @params, "console=tty" unless (get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/ || is_publiccloud);
```